.github/workflows/examples.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow is a demo of how to run all examples in the Icicle repository.
+# For each language directory (c++, Rust, etc.) the workflow 
+#   (1) loops over all examples (msm, ntt, etc.) and 
+#   (2) runs ./compile.sh and ./run.sh in each directory.
+# The script ./compile.sh should compile the example and ./run.sh should run it.
+# Each script should return 0 for success and 1 otherwise.
+
+name: Examples
+
+on:
+  push:
+    branches: temp/new-examples-api
```

Comment:
Update to run on the interval we want. @LeonHibnik

Reply:
I will use
```
on:
  pull_request:
    branches:
      - main
      - dev
  push:
    branches:
      - main
      - dev
      - temp/*

```

Reply:
done